<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoneAnnotator</class>
 <widget class="QMainWindow" name="BoneAnnotator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BoneAnnotator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="listWidget_ImageList">
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ScribbleArea" name="widget_ScribbleArea" native="true"/>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_BoneList">
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_OpenDirectory"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="action_OpenDirectory">
   <property name="text">
    <string>Open Directory...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ScribbleArea</class>
   <extends>QWidget</extends>
   <header>scribblearea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
